gmail/Module.md

```diff
@@ -1,54 +1,30 @@
-Connects to Gmail from Ballerina.
+## Overview
+Ballerina Gmail Connector provides the capability to send, read and delete emails through the Gmail REST API. It also provides the ability to read, trash, untrash and delete threads, ability to get the Gmail profile and mailbox history, etc. The connector handles OAuth 2.0 authentication.
 
-## Module Overview
+This module supports [Gmail API v1](https://developers.google.com/gmail/api).
 
-Ballerina Gmail Connector provides the capability to send, read and delete emails through the Gmail REST API. It also provides the ability to read, trash, untrash and delete threads, ability to get the Gmail profile and mailbox history, etc. The connector handles OAuth 2.0 authentication.
+## Prerequisites
 
-## Configurations
+Before using this connector in your Ballerina application, complete the following:
 
-Instantiate the connector by giving authentication details in the Gmail client config, which has built-in support for OAuth 2.0. Gmail uses OAuth 2.0 to authenticate and authorize requests. The Gmail connector can be minimally instantiated in the Gmail client config using the Access Token or by using the Client ID, Client Secret and Refresh Token.
+1. Create a [Google account](https://accounts.google.com/signup/v2/webcreateaccount?utm_source=ga-ob-search&utm_medium=google-account&flowName=GlifWebSignIn&flowEntry=SignUp). (If you already have one, you can use that.)
 
-**Obtaining Tokens to Run the Sample**
+2. Obtain tokens - Follow [this guide](https://developers.google.com/identity/protocols/oauth2)
```

Comment:
Take the description of point 2, to the next line so that it appears under point 2.

Reply:
Fixed by ddd90073069938a3660747b110c4643c53e57b4f